faq.md

```diff
@@ -133,6 +133,7 @@ import 'dart:io';
 import 'package:http/io_client.dart' as http;
 
 final ioc = new HttpClient();
+httpx.findProxy = (url) => 'PROXY 192.168.0.102:9090';
```

Comment:
There's an extra 'x' here. Remove it and the PR will be good to go. 😉

Reply:
upss, im sorry, 😆

Reply:
No harm done, thanks for the contribution! 😉 I also made a mistake telling you that it should be `http`, but in fact it should have been `ioc` as you fixed it. 😅